app/services/notifications/setup/download-recipients.service.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+/**
+ * Orchestrates fetching and formatting the data needed for the notifications setup download link
+ * @module DownloadRecipientsService
+ */
+
+/**
+ * Orchestrates fetching and formatting the data needed for the notifications setup download link
+ *
+ * The service will fetch recipients and then create a csv with the recipients' data.
+ * The service will return the name of the file and file type.
+ *
+ * @returns {Promise<object>} The data for the download link
+ */
+async function go() {
+  const csv = 'Licences\n12234\n'
+
+  return {
+    data: csv,
+    type: 'text/csv',
+    filename: 'recipients.csv'
```

Comment:
Are we happy with this name ? 

Do we need to add the due date / sleected returns period ? 

`summer-2022-03-01-recipients.csv` ?

Reply:
Updated